CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -70,10 +70,10 @@ public function buildQuickForm() {
       $this->addRule("value[$i]", ts('Please enter a valid money value (e.g. %1).', [1 => CRM_Utils_Money::formatLocaleNumericRoundedForDefaultCurrency('99.99')]), 'money');
 
       // default
-      $default[] = $this->createElement('radio', NULL, NULL, NULL, $i);
+      $default[$i] = NULL;
     }
 
-    $this->addGroup($default, 'default');
+    $this->addGroup('default', '', $default);
```

Comment:
This is causing an error when creating a new contribution page. Is there a reason the params are reversed?

Reply:
This is due to wrong function being called should be addRadio will fix shortly

Reply:
should be sorted now